src/Core/src/Handlers/Image/ImageHandler.Android.cs

```diff
@@ -47,8 +47,13 @@ public static void MapBackground(IImageHandler handler, IImage image)
 		public static void MapSource(IImageHandler handler, IImage image) =>
 			MapSourceAsync(handler, image).FireAndForget(handler);
 
-		public static Task MapSourceAsync(IImageHandler handler, IImage image) =>
-			handler.SourceLoader.UpdateImageSourceAsync();
+		public static Task MapSourceAsync(IImageHandler handler, IImage image)
+		{
+			return handler
+				.SourceLoader
+				.UpdateImageSourceAsync()
+				.ContinueWith((action) => handler.PlatformView.UpdateIsAnimationPlaying(image));
```

Comment:
```suggestion
				.ContinueWith((action) => handler.UpdateValue(nameof(IImage.IsAnimationPlaying)));
```

This will allow people to override the mapper and handle this case too. I think based on all the PRs that @PureWeen  has done, I think calling mappers is the best approach.